admin/configuration/default-interface.md

```diff
@@ -0,0 +1,27 @@
+# Default user interface
```

Comment:
For my own mental model - is this `configuration/` section meant to be more of a "reference" section in the docs framework we're using?

Reply:
@choldgraf yeah, I agree - this should be called 'configuration reference'.